app/styles/_notifications.less

```diff
@@ -37,6 +39,9 @@ notification-drawer-wrapper {
       display: block !important;
     }
   }
+  .drawer-pf-action {
```

Comment:
this seems like a change that should be made upstream, no?

Reply:
Yup, agree.  Looking into opening up a PR that way, then can either drop or close this.